articles/client-platforms/react/06-rules.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Rules
+description: This tutorial will show you how to use Auth0 rules to extend what Auth0 has to offer.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/06-Rules',
+}) %>_
+
+<%= include('../_includes/_rules-introduction') %>
+
+
+### Test rule result
```

Comment:
@alemata do you think we can follow the titles format from other tutorials? I'm always numerating and using a h2, something like `## 1. title`